tests/suite-integration/Entity.php

```diff
@@ -33,30 +33,31 @@
 
 use Glpi\Test\CommonTestCase;
 
-class Entity extends CommonTestCase {
+class Entity extends CommonTestCase
+{
 
-   public function beforeTestMethod($method) {
+   public function setUp() {
```

Comment:
Hi

I guess you renamed the method to speed up tests. The d rawback is if a test fails by generating a log entry in sql-errors or php-errors, then all subsequent tests will fail. Also, isolation between tests is less efficient as data in DB from a test may impact next ones in the same file.

Reply:
yes, the tests speeded up a lot of minutes after that. I'm fixing the errors, triying to have the most possible insolated scenarios and be able to run a single test as many times as posible to have always a success result.

Reply:
If this requires rewriting a lot of tests, you may consider to rewrite t hem in a "more unit" way, and less functional (suite-integration) at the same time. This might be a good opportunity.